infra/examples/aws-google-workspace/main.tf

```diff
@@ -194,55 +207,64 @@ module "worklytics-psoxy-connection" {
 
 locals {
   oauth_long_access_connectors = {
-    slack = {
-      deploy = true
-      function_name = "psoxy-slack-discovery-api"
-      source_kind = "slack"
+    slack-discovery-api = {
+      enabled           = true
+      source_kind       = "slack"
+      display_name      = "Slack Discovery API Connector"
       example_api_calls = []
+
     },
     zoom = {
-      deploy = true
-      function_name = "psoxy-zoom"
-      source_kind = "zoom"
+      enabled           = true
+      source_kind       = "zoom"
       example_api_calls = ["/v2/users"]
+      display_name      = "Zoom Connector"
     }
   }
+  enabled_oauth_long_access_connectors = { for id, spec in local.oauth_long_access_connectors : id => spec if spec.enabled }
+
 }
 
 # Create secret (later filled by customer)
 resource "aws_ssm_parameter" "long-access-token-secret" {
-  for_each = {
-  for k, v in local.oauth_long_access_connectors:
-  k => v if v.deploy
-  }
-  name        = "PSOXY_ACCESS_TOKEN_${each.value.function_name}"
```

Comment:
Review other usages of `PSOXY_ACCESS_TOKEN_` in the code. I think is in some markdown files and as well as in `modules/gcp-oauth-long-access-strategy/main.tf`

Reply:
I think it's OK:
![Cursor_and_psoxy_–_OAuthAccessTokenSourceAuthStrategy_java__core_](https://user-images.githubusercontent.com/102831/161195081-96465989-0c89-42f4-b088-a83f3f090d22.png)

`configService` will be a composite of namespaced config, which checks for the config property prefixed by namespace (`PSOXY_{function_name}_`) and then, if that's a miss, just the config property itself. 

The goal of that is to have all config properties pertaining to a particular lambda/function listed together.